Source/Core/DiscIO/Blob.cpp

```diff
@@ -177,23 +178,24 @@ std::unique_ptr<IBlobReader> CreateBlobReader(const std::string& filename)
   if (cdio_is_cdrom(filename))
     return DriveReader::Create(filename);
 
-  if (!File::Exists(filename))
+  File::IOFile file(filename, "rb");
+  u32 magic;
```

Comment:
Thank you @lioncash !  @JosJuice maybe calling it file_format_identifier or file_format_id would be more descriptive?  A comment calling this out as a magic number might also be useful?  Agree with @Helios747 , not terribly important but just an idea.